pivy-importer/src/main/groovy/com/linkedin/python/importer/deps/SdistDownloader.groovy

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2016 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.linkedin.python.importer.deps
+
+import com.linkedin.python.importer.distribution.SourceDistPackage
+import com.linkedin.python.importer.ivy.IvyFileWriter
+
+import groovy.util.logging.Slf4j
+import java.nio.file.Paths
+
+@Slf4j
+class SdistDownloader extends DependencyDownloader {
+    static final String SOURCE_DIST_PACKAGE_TYPE = "sdist"
+    static final String SOURCE_DIST_ORG = "pypi"
+
+    DependencySubstitution dependencySubstitution
+    boolean latestVersions
+    boolean allowPreReleases
+
+    SdistDownloader(String project, File ivyRepoRoot, DependencySubstitution dependencySubstitution,
+                    boolean latestVersions, boolean allowPreReleases, boolean lenient) {
+
+        super(project, ivyRepoRoot, lenient)
+        this.dependencySubstitution = dependencySubstitution
+        this.latestVersions = latestVersions
+        this.allowPreReleases = allowPreReleases
+    }
+
+    @Override
+    def downloadDependency(String dep) {
+        log.info("Pulling in $dep")
+        def (String name, String version) = dep.split(":")
```

Comment:
Not sure if it really matters, but in the original code, `name` and `version` were not typed. Is this better Groovy style?

Reply:
Intellij just keep complaining about this unclear type thing, so I changed it.

Reply:
Sounds good.